### DISCUSSION TITLE
Exporting labels

### DISCUSSION THREAD

**Discussion by 1Dr490n** [hlorenzi/customasm#202] [Ideas]
Hi! I would love to be able to export labels. For example, if I create a file that declares several labels (both by literals and by position in memory), it'd be great if the assembler could emit a second file with all the lables declared and with the correct values. That way, you could reuse code sections in another binary and you wouldn't have to assemble all at once. This could be very useful for libraries.

**Comment by hlorenzi** [comment]
Have a look at the `-f symbols` output option! Is this what you had in mind?

**Reply by 1Dr490n** [reply]
Yes, thank you!
